operator/cmd/mesh/manifest-common.go

```diff
@@ -167,36 +45,24 @@ func ignoreError(stderr string) bool {
 	return trimmedStdErr == ""
 }
 
-// fetchInstallPackageFromURL downloads installation packages from specified URL.
-func fetchInstallPackageFromURL(mergedIOPS *v1alpha1.IstioOperatorSpec) error {
-	if util.IsHTTPURL(mergedIOPS.InstallPackagePath) {
-		pkgPath, err := fetchInstallPackage(mergedIOPS.InstallPackagePath)
-		if err != nil {
-			return err
-		}
-		// TODO: replace with more robust logic to set local file path
-		mergedIOPS.InstallPackagePath = filepath.Join(pkgPath, helm.ChartsFilePath)
-	}
-	return nil
-}
-
-// fetchInstallPackage downloads installation packages from the given url.
-func fetchInstallPackage(url string) (string, error) {
-	uf, err := helm.NewURLFetcher(url, "")
+// yAMLFromSetFlags takes a slice of --set flag key-value pairs and returns a YAML tree representation.
+// If force is set, validation errors cause warning messages to be written to logger rather than causing error.
+func yAMLFromSetFlags(setOverlay []string, force bool, l *Logger) (string, error) {
```

Comment:
nit :per @fpesce complaining to me this morning about this, this should be `yamlFromSetFlags`

Reply:
c.f. [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". As an example: ServeHTTP not ServeHttp. For identifiers with multiple initialized "words", use for example "xmlHTTPRequest" or "XMLHTTPRequest".

This rule also applies to "ID" when it is short for "identifier" (which is pretty much all cases when it's not the "id" as in "ego", "superego"), so write "appID" instead of "appId".

Code generated by the protocol buffer compiler is exempt from this rule. Human-written code is held to a higher standard than machine-written code.

Reply:
thanks for reminding me... once in a while i realize that i haven't been following the all caps rule and put in one of these, because i forget about the supplementary rule in this case :-)